patches/747-mikrotik-extra-support.patch

```diff
@@ -552,44 +573,48 @@
 +#include "qcom-ipq4018-sxtsq-5-ac.dts"
 +
 +/ {
-+	compatible = "mikrotik,ldf-5acd";
++ compatible = "mikrotik,ldf-5acd";
 +	model = "MikroTik LDF 5 ac (RBLDFG-5acD)";
 +};
 --- a/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
 +++ b/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
-@@ -210,6 +210,8 @@
+@@ -210,6 +210,9 @@
  	mikrotik,hap-ac3 |\
  	mikrotik,hap-ac3-lte6-kit |\
  	mikrotik,sxtsq-5-ac |\
++	mikrotik,discg-5acd |\
 +	mikrotik,lhgg-5acd* |\
 +	mikrotik,ldf-5acd |\
  	mikrotik,wap-ac |\
  	mikrotik,wap-ac-lte |\
  	mikrotik,wap-r-ac)
-@@ -249,6 +251,8 @@
+@@ -249,6 +251,9 @@
  	mikrotik,hap-ac2 |\
  	mikrotik,hap-ac3 |\
  	mikrotik,hap-ac3-lte6-kit |\
++	mikrotik,discg-5acd |\
 +	mikrotik,lhgg-5acd* |\
 +	mikrotik,ldf-5acd |\
  	mikrotik,wap-ac |\
  	mikrotik,wap-ac-lte |\
  	mikrotik,wap-r-ac)
-@@ -265,6 +269,8 @@
+@@ -265,6 +269,9 @@
  	mikrotik,hap-ac3 |\
-         mikrotik,hap-ac3-lte6-kit |\
+ 	mikrotik,hap-ac3-lte6-kit |\
```

Comment:
Change because of whitespace - please revert it.

Reply:
reverted. Updated version does show a line removed and identical line added.